asapdiscovery-ml/asapdiscovery/ml/cli.py

```diff
@@ -0,0 +1,467 @@
+import json
+from pathlib import Path
+
+import asapdiscovery.ml.schema_v2.config as ascfg
+import click
+
+
+@click.group()
+def ml():
+    pass
+
+
+@ml.command()
+@click.option(
```

Comment:
You can use the pre-packaged one from `asapdiscovery-cli` or just keep this one if the defaults don't suit.

Reply:
I think the issue I had with that one was that I needed it to be required. is there a way to modify a click option after importing it? alternatively, would it make sense to have an `output_dir_required` function in `asapdiscovery-cli` that's the same as `output_dir` but with the `required` flag set?

Reply:
Its fine tbh, its okay to repeat, as long as same API preserved.